config/jobs/kubernetes-sigs/cluster-api-provider-vsphere/cluster-api-provider-vsphere-presets.yaml

```
@@ -179,3 +179,48 @@ presets:
         path: ca.crt
       - key: vmc-vpn-tls.key
         path: tls.key
+- labels:
+    # For kubernetes-sigs/cluster-api-provider-vsphere janitor
+    preset-janitor-vsphere-config: "true"
```

```suggestion
    preset-vsphere-janitor-config: "true"
```
if there is no a prior art, I think we should have vsphere first